src/Forms/EffectKaraoke.cs

```diff
@@ -36,7 +36,11 @@ private void FormEffectkaraoke_KeyDown(object sender, KeyEventArgs e)
 
         internal void Initialize(Paragraph paragraph)
         {
-            _paragraph = paragraph;
+            _paragraph = new Paragraph(paragraph);
+            if (_paragraph.Text.Length > 3)
+            {
+                _paragraph.Text = HtmlUtil.FixUpperTags(_paragraph.Text);
+            }

```

Comment:
Do all change in a copy instead (Allows us to revert if tags was indeed lowered)